### PR TITLE
Open up the prerelease registry to non-workers-sdk repos

### PR DESCRIPTION
**Pull request by GregBrimble**
What this PR solves / how to test:

I want to use the prerelease registry for things outside of just `workers-sdk`! So this opens it up to all Cloudflare repos :)

Note, this (intentionally) breaks any existing URLs. To fix, re-run the Action.

Reviewer has performed the following, where applicable:

- [ ] Checked for inclusion of relevant tests
- [ ] Checked for inclusion of a relevant changeset
- [ ] Checked for creation of associated docs updates
- [ ] Manually pulled down the changes and spot-tested

### CODE REVIEW

**Review by JacobMGEvans — APPROVED** [review]
Interested in what other repos you were wanting to use this in?

**Review by lrapoport-cf — DISMISSED** [review]
@GregBrimble this is probably something the team should discuss to better understand the impact before making a decision on landing or not. please hold off for now

**Comment by lrapoport-cf** [comment]
> Thanks for adding the allowlist and expiration comment!

ditto, thanks @GregBrimble  for addressing devprod team feedback!

**Pull request merged**
